clients/spark/build.sbt

```
@@ -1,18 +1,10 @@
 import build.BuildType
 
 lazy val baseName = "lakefs-spark"
+lazy val projectVersion = "0.8.2"
```

```suggestion
lazy val projectVersion = "0.9.0"
```

This is a "major-y" change: some existing installations will no longer get artefacts!  Luckily we're pre-1.0.0 so semver says (or, rather, semver < 2.0.0 says) we just need to bump the most significant nonzero number.